tools/cldr/cldr-to-icu/src/main/java/org/unicode/icu/tool/cldrtoicu/IcuData.java

```diff
@@ -155,16 +113,6 @@ private RbValue replacePathValues(String value) {
         return Collections.unmodifiableSet(paths);
     }
 
-    /** Returns whether the given path is present in this instance. */
```

Comment:
Some of these methods were used exactly once, so I inlined them to reduce the surface area.